client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -66,16 +70,27 @@ export const CommentDetailHeader = ( {
 					/>
 				</label>
 			}
-			<div className="comment-detail__author-info">
+			<div className="comment-detail__author-preview">
 				<div className="comment-detail__author-avatar">
 					<img className="comment-detail__author-avatar-image" src={ authorAvatarUrl } />
 				</div>
-				<strong>
-					{ authorDisplayName }
-				</strong>
-				<span>
-					{ authorUrl }
-				</span>
+				<div className="comment-detail__author-info">
+					<div className="comment-detail__author-info-element">
+						<strong>
+							{ authorDisplayName }
+						</strong>
+						<span>
+							{ authorUrl }
+						</span>
+					</div>
+					<div className="comment-detail__author-info-element">
+						{ translate( 'on {{postLink/}}', {
+							components: {
+								postLink: <a href={ postUrl }>{ postTitle }</a>,
```

Comment:
Minor, but I noticed that clicking on this expands the comment, before redirecting to the site.